velox/exec/RowNumber.cpp

```diff
@@ -74,14 +77,14 @@ void RowNumber::addInput(RowVectorPtr input) {
   input_ = std::move(input);
 }
 
-FlatVector<int64_t>& RowNumber::getOrCreateRowNumberVector(vector_size_t size) {
+VectorPtr RowNumber::getOrCreateRowNumberVector(vector_size_t size) {
```

Comment:
Why is this change?

Reply:
At first I changed to the type to a pointer type and after review to the type you commented on. My original idea was so it can return a value that can be easily assigned (reference types need to be initialized and this wouldn't have worked).
But, the original function returns an l-value and I can take the address of that and assign to a pointer type variable. I reverted the change and adjusted accordingly.